README.md

```diff
@@ -138,6 +138,10 @@ our company and culture evolve, so should this document.
 repeating the mistakes of the past.
 - Ensure you challenge designs that you see problems with instead of implementing them without questioning.
 
+#### Spread knowledge
+- Create documentation for the software you write (sometimes [even before you write it](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html))
```

Comment:
So my feeling about this point is that it belongs in "Communication" rather than "Always Grow". We actually also have a section about writing "Minimum Viable Documentation" - any thoughts on how we can consolidate those two points? Do they need consolidation?